apps/src/music/player/sequencer/MusicPlayerStubSequencer.ts

```diff
@@ -0,0 +1,34 @@
+import {PlaybackEvent} from '../interfaces/PlaybackEvent';
+import Sequencer from './Sequencer';
+
+/**
+ * A stubbed out version of {@link Sequencer} meant to be used by block
+ * modes that use the MusicPlayer API. This allows us migrate block modes
+ * over to using the Sequencer model as we like, without completely crashing
+ * if a block mode does not yet have a sequencer implementation.
+ */
+export default class MusicPlayerStubSequencer extends Sequencer {
+  playSoundAtMeasureById() {
+    console.log('Not implemented');
+  }
+
+  createTrack() {
+    console.log('Not implemented');
+  }
+
+  addSoundsToTrack() {
+    console.log('Not implemented');
+  }
+
+  addRestToTrack() {
+    console.log('Not implemented');
+  }
```

Comment:
What is a "track" here?  Is it specific to the "tracks" model?

Reply:
Yep, these are just stubs for the existing track functions in MusicPlayer. The blockly code in tracks model can use these until a TracksSequencer is created.

Reply:
Makes sense!